homeassistant/components/calendar/__init__.py

```diff
@@ -850,6 +858,20 @@ async def async_create_event(entity: CalendarEntity, call: ServiceCall) -> None:
 
 async def async_list_events_service(
     calendar: CalendarEntity, service_call: ServiceCall
+) -> ServiceResponse:
+    """List events on a calendar during a time range.
+
+    Deprecated: please use async_events_service.
+    """
+    _LOGGER.warning(
+        "Detect use of service 'calendar.list_events'. This is deprecated and will stop working in Home Assistant 2024.6",
```

Comment:
Does it make sense to say something like "Use `calendar.events` which supports multiple entities" instead?

Reply:
What do you think about `calendar.get_events`?

Reply:
We should refer to what the user should use instead but personally I think `get_events` is a better fit for a new service.
Also it should raise an issue and not just logging.

Reply:
I just had a quick look through the code base. Can this automatically be repaired? I would have to go through all automations,scripts,intents,... and replace the service name